index.mjs

```diff
@@ -32,6 +33,7 @@ export const globby = Object.assign(function globby(...args) {
   return globbyModule.globby(...args)
 }, globbyModule)
 export const glob = globby
+export const yaml = { parse: yamljs.parse, stringify: yamljs.stringify }
```

Comment:
Let's export everything. And let's use `YAML` same as `JSON`.

Reply:
✅